Applications/CMC/test/testCMCArm26_Millard.cpp

```diff
@@ -45,15 +45,15 @@ void testCMCArm26() {
     Storage *standard = new Storage();
     cmc.getModel().formStateStorage(temp, *standard);
 
-    Array<double> rms_tols(0.02, 2*2+2*6); // activations within 2%, angles within .6 degrees
+    Array<double> rms_tols(0.015, 2*2+2*6); // activations within 1%, angles within .6 degrees
```

Comment:
It would be ideal if one tolerance could be set here (say 2%) and the individual tolerances not being adjusted below (as in the Thelen case). If that is not possible then this fine. Should say 1.5% in the comment though.